services/spark/helm/spark/templates/spark-master-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-spark-debug-config
```

Comment:
1. Should be `-spark-master-config` no?
2. Also, should be in _helpers

Reply:
1. True. Fixing it.
2. Will do.